transform/emr_transform_news.py

```
@@ -40,9 +52,96 @@ def transform(data_source:str, output_uri:str, batch_period:str)-> None:
+        dbname= conf.RDS_PROPERTY["dbname"], 
+        user=conf.RDS_PROPERTY["user"], 
+        password=conf.RDS_PROPERTY["password"],
+        host=conf.RDS_PROPERTY["url"],
+        port=conf.RDS_PROPERTY["port"], 
+    )
+    cur = conn.cursor()
+
+    query = f"""
+        UPDATE accumulated_table
+        SET is_alert = FALSE;
+    """
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)